woocommerce/woocommerce
# CYS - set flowType when the state machine is instantiated

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:








### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure that you are on a WooCommerce site where the CYS AI flow works.
2. Open the network tools.
3. Head over to WooCommerce > Home > Customize your store.
4. Ensure that the `status.json` is fetched.
5. Click on "Design With AI" and proceed to the Pattern Assembler.
6. Ensure that the `status.json` isn't fetched again. You should see just one entry:

![image](https://github.com/woocommerce/woocommerce/assets/4463174/5c70533a-6a35-4916-93a6-eac0c674ffc2)

7. Smoke test the CYS flow.





### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

CYS - set flowType when the state machine is instantiated.

</details>
